lib/matplotlib/backends/backend_ps.py

```diff
@@ -184,10 +184,10 @@ def _font_to_ps_type3(font_path, chars):
         g = font.load_glyph(glyph_id, LOAD_NO_SCALE)
         v, c = font.get_path()
         entries.append(
-            "/%(name)s{%(bbox)s sc\n" % {
-                "name": font.get_glyph_name(glyph_id),
-                "bbox": " ".join(map(str, [g.horiAdvance, 0, *g.bbox])),
-            }
+            "/{name}{{{bbox} sc\n".format(
+                name=font.get_glyph_name(glyph_id),
+                bbox=" ".join(map(str, [g.horiAdvance, 0, *g.bbox])),
+            )
```

Comment:
I *think* that one can get f-strings here as well (although those will be horrible...) if it is executed again?